sys/kern/subr_module.c

```diff
@@ -427,8 +427,9 @@ preload_modinfo_type(struct sbuf *sbp, int type)
 static void
 preload_modinfo_value(struct sbuf *sbp, uint32_t *bptr, int type, int len)
 {
-#ifdef __LP64__
-#define sbuf_print_vmoffset(sb, o)	sbuf_printf(sb, "0x%016lx", o);
+#if defined(__LP64__) || __has_feature(capabilities)
```

Comment:
```suggestion
#if __SIZEOF_SIZE_T__ == 8
```

Reply:
I was not aware of the `__SIZEOF_SIZE_T__` define, if we think this is the best way perhaps we should replace most of the `__LP64__` guards with it?

Reply:
Yeah, most of them should be that or long, and the ones that really care about P64 should use `__SIZEOF_POINTER__` instead.